### PR TITLE
Add ChosenNews and GDG Douala

### PR DESCRIPTION
**Pull request by DipandaAser**
Related to #1

### CODE REVIEW

File: src/res/apps.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ items:
       - https://blog.osscameroon.com
       - https://miniyotas.osscameroon.com
     image: default.svg
-#--------------------------------------------------------------
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:35]: Were this indentations changes done in purpose ?

File: src/res/apps.yaml [hunk 2]
```diff
@@ -91,21 +91,21 @@ items:
...
   - title: "Le Responsable Mboa"
     category: video game
     #max of five tags
     tags:
       - social
       - jeux videos
     description: |
-     Sous très haute instruction de Son Excellentissime Mougabiang, vous allez enfin pouvoir vivre une première semaine en tant que stagiaire dans le "Ministère Général", dans ce jeu de gestion/simulation de vie satyrique {ENCORE EN DEVELOPPEMENT}.
+      Sous très haute instruction de Son Excellentissime Mougabiang, vous allez enfin pouvoir vivre une première semaine en tant que stagiaire dans le "Ministère Général", dans ce jeu de gestion/simulation de vie satyrique {ENCORE EN DEVELOPPEMENT}.
```
Comment (elhmn) [inline src/res/apps.yaml:102]: Ditto

File: src/res/apps.yaml [hunk 3]
```diff
@@ -312,4 +312,36 @@ items:
     #facebook link
     facebook: https://www.facebook.com/groups/devcdouala/
     image: default.svg
+  #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:315]: I believe this comment should start at beginning of the line

**Review by elhmn — COMMENTED** [review]
Thank you for your contribution 🙇‍♂️ , I just left few comments related to the indentation and formatting changes you have made to the file.

**Comment by DipandaAser** [comment]
@elhmn my IDE format the file when i saved.
I made another commit to fix the indentation.

**Comment by elhmn** [comment]
> @elhmn my IDE format the file when i saved.
> I made another commit to fix the indentation.

Yeah, I guessed so 😄 . I just checked but it seems that your PR still contains changes on other lines than the ones you intended to change or add.

Can you please fix that ?

**Comment by DipandaAser** [comment]
@elhmn Now it is good

**Pull request merged**